pkg/cluster/arooperator.go

```diff
@@ -65,6 +65,10 @@ func (m *manager) ensureCredentialsRequest(ctx context.Context) error {
 	return m.aroOperatorDeployer.CreateOrUpdateCredentialsRequest(ctx)
 }
 
+func (m *manager) ensureUpgradeAnnotations(ctx context.Context) error {
```

Comment:
```suggestion
func (m *manager) ensureUpgradeAnnotation(ctx context.Context) error {
```

Suggesting this change for clarity's sake since there's only one annotation.